### PR TITLE
Update go client and fix validation pipeline

### PR DESCRIPTION
**Pull request by ImJeremyHe**

### CODE REVIEW

File: arbitrator/vid-helper/src/bytes.rs [hunk 1]
```diff
@@ -0,0 +1,160 @@
+//! Convenient serialization for binary blobs.
```
Comment (sveitser) [inline arbitrator/vid-helper/src/bytes.rs:1]: Can use https://github.com/EspressoSystems/base64-bytes now

File: arbitrator/vid-helper/src/namespace.rs [hunk 2]
```diff
@@ -283,18 +286,21 @@ impl NamespaceProof {
...
 pub struct Transaction {
-    _vm: VmId,
+    _namespace: NamespaceId,
     pub payload: Vec<u8>,
 }
 
 impl Transaction {
-    pub fn new(vm: VmId, payload: Vec<u8>) -> Self {
-        Self { _vm: vm, payload }
+    pub fn new(vm: NamespaceId, payload: Vec<u8>) -> Self {
```
Comment (sveitser) [inline arbitrator/vid-helper/src/namespace.rs:294]: Remove this `vm` too?

**Review by sveitser — APPROVED** [review]
LGTM, minor comments. We can remove the copied base 64 bytes from this repo and use our standalone crate.

File: arbitrator/vid-helper/src/bytes.rs (hunk 1 above)
Comment (ImJeremyHe) [reply arbitrator/vid-helper/src/bytes.rs:1]: This crate seems not have the full functionalities required here

**Pull request merged**